lib/ng-directive-compiler-helper.js

```diff
@@ -32,6 +32,12 @@ function createCompiler(template, $rootScope, $compile) {
 
     callback(extendedScope.$$childHead, element);
 
-    return { scope: extendedScope.$$childHead, element: element};
+    function andGet(elementTag) {
+      var childElement = element.find(elementTag).children(0);
+      var childScope = childElement.scope();
```

Comment:
@koriguy this will fail if `elementTag` is not found on `element`. Need to fallback to something, maybe even a log message telling about it?